Makefile

```diff
@@ -368,7 +357,7 @@ ggml-cuda.o: ggml-cuda.cu ggml-cuda.h
 endif # LLAMA_HIPBLAS
 
 ifdef LLAMA_METAL
-	MK_CPPFLAGS += -DGGML_USE_METAL #-DGGML_METAL_NDEBUG
+	MK_CPPFLAGS += -DGGML_USE_METAL
```

Comment:
It's come from https://github.com/ggerganov/llama.cpp/commit/3a007648f230ea37d6cca5e63850f04ebb12d2cf.

@ggerganov Can we have a flag like `LLAMA_METAL_NDEBUG` for that? I use this flag when using tool like llama-bench, other times I think it would be useful not to enable the flag.

Reply:
Yes, we can add `LLAMA_METAL_NDEBUG` to optionally disable metal debug prints.
I still prefer to have them enabled by default, but in the future, when things are more stable, we will disable them by default.